src/rabbit_control_main.erl

```diff
@@ -519,6 +519,9 @@ action(list_vhosts, Node, Args, _Opts, Inform, Timeout) ->
     display_info_list(call(Node, {rabbit_vhost, info_all, []}, Timeout),
                       ArgAtoms);
 
+action(list_user_permissions, _Node, _Args = [], _Opts, _Inform, _Timeout) ->
```

Comment:
wouldn't the empty string be a problem with most commands actually? I think for that case we get an error like this:

```
Listing permissions for user [] ...
Error: no_such_user: 
```

which at least tells me something about the problem, not like `function_clause` before